src/tokenized-line.coffee

```diff
@@ -41,10 +41,20 @@ class TokenizedLine
   copy: ->
     new TokenizedLine({@tokens, @lineEnding, @ruleStack, @startBufferColumn, @fold})
 
+  # This clips a given screen column to a valid column that's within the line
+  # and not in the middle of any atomic tokens.
+  #
+  # column - A {Number} representing the column to clip
+  # options - A hash with the key clipToAtomicTokenEdge. Valid values for this key:
+  #           'closest' (default): clip to the closest edge of an atomic token.
+  #           'right': clip to the right edge.
+  #           'left': clip to the left edge.
+  #
+  # Returns a {Number} representing the clipped column.
   clipScreenColumn: (column, options={}) ->
     return 0 if @tokens.length == 0
 
-    { skipAtomicTokens } = options
+    { clipToAtomicTokenEdge } = options
```

Comment:
What do you guys think about just calling this `clip` and doing `forward` or `backward` as option values like we're doing in the new `text-document`?